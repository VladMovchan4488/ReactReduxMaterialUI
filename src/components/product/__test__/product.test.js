import React from "react";
import Product from "../product";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

it("Snapshot my Product", () => {
  const params = {
    bsr_category: "Home & Kitchen",
    price: 15.79,
    stars: 4.6,
    name: "A3 Magnetic Dry Wipe Weekly Planner Whiteboard for Fridge"
  };

  const wrapper = shallow(<Product {...params} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
