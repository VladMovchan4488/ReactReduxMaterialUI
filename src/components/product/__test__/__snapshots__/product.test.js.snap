// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot my Product 1`] = `
<WithStyles(Component)
  className="table"
  columns={
    Array [
      Object {
        "field": "name",
        "title": "Name",
      },
      Object {
        "field": "price",
        "title": "Price",
      },
      Object {
        "field": "stars",
        "title": "Stars",
      },
      Object {
        "field": "bsr_category",
        "title": "Category",
      },
      Object {
        "field": "brand",
        "title": "Brand",
      },
    ]
  }
  title="Products"
/>
`;
